src/index.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./styles.css" />
+    <title>PHP_CodeSniffer PHAR - Latest PHAR files</title>
+    <link rel="canonical" href="https://phars.phpcodesniffer.com/" />
+</head>
+
+<body>
+    <main class="container">
+        <h1 class="heading">Download the latest PHAR files</h1>
+        <div class="download-section">
+            <article class="download-box">
+                <h2 class="download-box__title">PHPCS</h2>
+                <div class="download-box__description">
+                    Tokenizes PHP, JavaScript and CSS files to detect violations of a
+                    defined coding standard.
+                </div>
+                <a href="phpcs.phar" download class="button button--download"><svg xmlns="http://www.w3.org/2000/svg"
+                        class="icon" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor"
+                        stroke-width="2" stroke-linecap="round" stroke-linejoin="round"
+                        class="feather feather-download">
+                        <path d="M21 15v4a2 2 0 0 1-2 2H5a2 2 0 0 1-2-2v-4"></path>
+                        <polyline points="7 10 12 15 17 10"></polyline>
+                        <line x1="12" y1="15" x2="12" y2="3"></line>
+                    </svg><span>Download PHPCS</span></a>
+                <a href="phpcs.phar.asc" class="download-box__secondary-link">Signature (.asc)</a>
+            </article>
+            <article class="download-box">
+                <h2 class="download-box__title">PHPCBF</h2>
+                <div class="download-box__description">
+                    Automatically corrects coding standard violations.
+                </div>
+                <a href="phpcbf.phar" download class="button button--download"><svg xmlns="http://www.w3.org/2000/svg"
+                        class="icon" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor"
+                        stroke-width="2" stroke-linecap="round" stroke-linejoin="round"
+                        class="feather feather-download">
+                        <path d="M21 15v4a2 2 0 0 1-2 2H5a2 2 0 0 1-2-2v-4"></path>
+                        <polyline points="7 10 12 15 17 10"></polyline>
+                        <line x1="12" y1="15" x2="12" y2="3"></line>
+                    </svg><span>Download PHPCBF</span></a>
+                <a href="phpcbf.phar.asc" class="download-box__secondary-link">Signature (.asc)</a>
+            </article>
+        </div>
+        <aside class="additional-info">
+            <a href="./phars/">
+                See all PHAR files
```

Comment:
We may need to think about this phrase a little.

The PHAR website doesn't contain all previously released PHAR files. Only the ones released since PHIVE support was added (and the PHAR files started to be signed).